src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -182,9 +193,10 @@ static void verify_continuation(oop continuation) { Continuation::debug_verify_c
 
 static void do_deopt_after_thaw(JavaThread* thread);
 static bool do_verify_after_thaw(JavaThread* thread, stackChunkOop chunk, outputStream* st);
-static void log_frames(JavaThread* thread);
-static void log_frames_after_thaw(JavaThread* thread, ContinuationWrapper& cont, intptr_t* sp, bool preempted);
+static void log_frames(JavaThread* thread, bool dolog = true);
```

Comment:
```suggestion
static void log_frames(JavaThread* thread, bool do_log = true);
```

Reply:
I removed it altogether. I used it during debugging when I wanted to disable all frame logging except from certain places. But it's not really needed.